The Deep (minor) Flavor Missions (#6234)
endless-sky/endless-sky: data/human/deep missions.txt

			`	"Yer that Cap'n that Skaldgar always went on about, aren'tcha?" The young man attempts to imitate Skaldgar with a deep and ponderous voice, "Him that knows the currents of space like we know our stone, and the like." The man grins sheepishly, " Ol' Skaldgar always had a way with words, he did."`
			`	The young man then looks down, somberly, and pulls from his pocket a small, well-worn stone with some strange markings on it and holds it out to you.`
			`	"He wanted," he begins haltingly, "he wanted you, only you, to take this back to his folk on Norn." He swallows. "It's the old ways, it is. This, it needs to be returned, and not by a stranger." The young man then shrugs. "Best to letcha know, I ain't have no money to pay you. This would have to be done as a favor for the ol' man."`
			choice
				`	(You reach out and accept the stone.)`
					goto accept
				`	"Nope. Not going to do it. I run a ship, not a charity."`
			`	The young roustabout nods knowingly. "Guess Skaldgar was wrong about you, but I understand. We all gotta eat." He turns and slides back into the endless crowd.`
				decline
			label accept
			`	You're surprised at how cool the smooth stone feels in your palm.`

> 			`	He smiles sadly. "Skaldgar always had a nose for good folks. Anyways, he told me he had a nephew named Helm who worked at the Norn spaceport. Find 'em and he can getcha and this stone where it needs to go." The young man than looks you in the eye and gives a slow respectful nod, "For Ol' Skaldgar, may he be findin' ore in hel."`

Reviewer: "Helm" seems a rather odd name, consider something that is not also a piece of headgear?

Author: Helm means helm. It's a pretty common Norse name/name element. 
Even in modern times some people have that time: 
https://en.wikipedia.org/wiki/Helm_(given_name)

I would prefer to keep it.

			`	Through all of this, you got a better feel for Thorleif, Skaldgard and his people, and were almost sorry when your ship approached Alfheim.`
			`	On landing, you are surprised by how much Alfheim reminds you of Helheim. Though the environment and ecology of the worlds are remarkably different, both are primarily used for resource extraction, and thus filled with boomtown employees and roustabouts, looking for work, chasing dreams, and trying to make their way in the world.`
			`	Thorleif points to the spaceport. "Let's see if we can find a guide in there. There's a village we'll need to find, assuming it even exists anymore and hasn't been abandoned or destroyed to make room for a new oil rig."`
	on offer
		conversation
			`Helm meets you in the nearly deserted spaceport bar. "Ma' always had a soft spot for Uncle Skaldgar, even after he ran off 35 years ago, she still spoke fondly of him. He always sent the family money, he was good like that. Not as good, though, in sending his love."`

> 			`	Helm sighs, "This is going to be tough, but as bearer of the Stone, you are going to need to present it to his son. His family lives on a floating island to the south of here. Are you willing to accompany me?"`

Reviewer: The description of Norn implies that much of the industry floats, but the islands are regular fixed landmasses. Using habitat felt a bit more appropriate (and gets rid of the mental image I got of an island floating in the air).
```suggestion
			`	Helm sighs, "This is going to be tough, but as bearer of the Stone, you are going to need to present it to his son. His family lives on a floating habitat to the south of here. Are you willing to accompany me?"`
```

			label behalf
			`	"Wait," you say before Thorleif can stomp off. "I do not, could not, know all that has happened here, that has gone on between you and your father. But I know that every time I met him he clearly regretted whatever happened, and constantly spoke of his mistakes. He is gone now, mistakes and all, and this stone.. I don't think it represents forgiveness, just, acceptance of who he was and that now he is gone."`
			`	Thorleif turns to you, his face flashing red, and speaks through clenched teeth, "I have no father. I have no memories of such a man, and I will make no effort to remember someone who didn't stick around long enough to know me. Now, go to your rooms and leave me be!"`
			label rooms
			`	You quietly follow Helm to two small, sparsely decorated rooms, clearly intended for guest use. Helm clears his throat, "This may surprise you, but that actually went better than I thought it would."`
			`	Helm then turns his attention to a small window and the darkening horizon beyond it. "Dark clouds are gathering, I fear we may be here longer than a single night."`
			choice
				`	(You head to bed as the first rains begin to patter against the roof.)`

> 			`	You awake to riotous rain pelting hard against the walls. As Helm predicted a powerful storm blew in over the night, and the floating island found itself besieged by savage weather. By midmorning the walls are starting to creak as the howling wind bashes against it unceasingly.`

Reviewer 2: ```suggestion
			`	You awake to riotous rain pelting hard against the walls. As Helm predicted, a powerful storm blew in over the night, and the floating island found itself besieged by savage weather. By midmorning the walls are starting to creak as the howling wind bashes against it unceasingly.`
```

> 			`	You hear a door open and struggle to close. Then the sound of heavy sloshing footsteps. Thorleif, soaked to the bone, stands before you and Helm in the main room, "It's a Fimbulhrithr," Thorleif exclaims, "One of the worst I've yet seen. If we do not act quickly it could well flatten everything on this isle. Will you help me?"`

Reviewer 2: ```suggestion
			`	You hear a door open and struggle to close, followed by the sound of heavy sloshing footsteps. Thorleif, soaked to the bone, stands before you and Helm in the main room. "It's a Fimbulhrithr," Thorleif exclaims, "One of the worst I've yet seen. If we do not act quickly it could well flatten everything on this isle. Will you help me?"`
```
Personal preference. With just the correction below:
```suggestion
			`	You hear a door open and struggle to close. Then the sound of heavy sloshing footsteps. Thorleif, soaked to the bone, stands before you and Helm in the main room. "It's a Fimbulhrithr," Thorleif exclaims, "One of the worst I've yet seen. If we do not act quickly it could well flatten everything on this isle. Will you help me?"`
```

			choice
				`	(You offer to help Thorleif.)`
					goto help

> 				`	("No, I am done too much for this family already. Especially for free!"`

Reviewer 2: ```suggestion
				`	"No, I am done too much for this family already. Especially for free!"`
```

> 			label refuse

Reviewer 2: ```suggestion
```

			`	Thorleif growls, "Knew better than to trust a friend of that useless old man. I gave you a place to sleep, and now you won't lift a hand to save it? I should have expected no less!" Thorleif and Helm hurry off to tend to the island during the storm, leaving you warm and cozy and idle inside.`

> 			`	Thorleif and Helm heoricially manage to steer the island away from the storm, and to calmer safer seas. However, with only the two of them many of the out-buildings are destroyed along with a few of the small watercraft Thorleif used to fish and work on.`

Reviewer 2: ```suggestion
			`	Thorleif and Helm heroically manage to steer the island away from the storm and to calmer, safer seas. However, with only the two of them many of the out-buildings are destroyed, along with a few of the small watercraft Thorleif used to fish and work on.`
```

> 			`	You get the impression that while Thorleif is not ruined, this is an expensive and major loss for him. The moment the winds are calm, he demands Helm take you back to the mainland, and your ship. "This Captain," Thorleif's voice oozing with disgust, "is just as lucky as my so-called father. I want him gone."`

Reviewer 2: ```suggestion
			`	You get the impression that while Thorleif is not ruined, this is an expensive and major loss for him. The moment the winds are calm, he demands Helm take you back to the mainland and your ship. "This Captain," Thorleif says, his voice oozing with disgust, "is just as lucky as my so-called father. I want him gone."`
```

> 			`	For most of the trip back to your ship Helm says nothing, he has been cool to you ever since you refused to help save the island. Finally, as you are preparing to disembark from his ship he stops you. "You've done Skaldgar no favors here. I'd appreciate it if we don't see from you again."`

Reviewer 2: ```suggestion
			`	For most of the trip back to your ship, Helm says nothing. He has been cool to you ever since you refused to help save the island. Finally, as you are preparing to disembark from his ship, he stops you. "You've done Skaldgar no favors here. I'd appreciate it if we don't see from you again."`
```

> 			`	Helm holds up his hand showing off the stone in his palm, "I took it while you were napping. Skaldgar made a mistake with you, and I will at least see the stone properly tended. Goodbye." Helm indicates towards the exit and waits for you to depart before cast off again in his ship.`

Reviewer 2: ```suggestion
			`	Helm holds up his hand showing off the stone in his palm. "I took it while you were napping. Skaldgar made a mistake with you, and I will at least see the stone properly tended. Goodbye." Helm indicates toward the exit and waits for you to depart before casting off again in his ship.`
```

				decline
			label help
			`	Thorleif nods and takes you and Helm outside into the storm, or Fimbulhrithr. Thorleif and Helm work the oars to steer the island away from the storm, while you are given the task of tying everything down that is liable to fly away.`

> 			`	With the storm comes unnatural darkness, and the sky is filled with debris and fish and all manner of items that do not belong in it. You hurry as fast as you can, saving as much as you can.`

Reviewer 2: ```suggestion
			`	With the storm comes unnatural darkness, and the sky is filled with debris and fish and all manner of items that do not belong in it. You hurry as fast as you can, saving as much as possible.`
```
The repetition of `as you can` doesn't really land for me here. Your choice.

> 			`	After hours of hard work, the storm begins to subside and the sea and sky begin to calm. An exhausted Thorleif and Helm stagger over to you, and Thorleif surveys his small island. "Without your help I would have lost many of the outbuildings. It was a lucky thing you came when you did."`

Reviewer 2: ```suggestion
			`	After hours of hard work, the storm begins to subside, and the sea and sky begin to calm. An exhausted Thorleif and Helm stagger over to you, and Thorleif surveys his small island. "Without your help I would have lost many of the outbuildings. It was a lucky thing you came when you did."`
```

> 			`	With that he turns and heads back to the main dwelling, with you and Helm in tired tow.`

Reviewer 2: ```suggestion
			`	With that, he turns and heads back to the main dwelling, with you and Helm in tired tow.`
```
`Tired tow`'s a nice little alliteration! Rolls off the tongue quite well.

			choice
				`	(Wash up and warm up in the main room around the fire.)`

> 					goto fire
> 			label fire

Reviewer 2: ```suggestion
```

> 			`	 You spend the evening warming yourself by a roaring fire telling Helm and Thorleif tales of your exploits in space, and many of the wonders you've seen.`

Reviewer 2: ```suggestion
			`	 You spend the evening warming yourself by a roaring fire, telling Helm and Thorleif tales of your exploits in space and many of the wonders you've seen.`
```

> 			`	In turn they tell you about the sea beasts, leviathans, and krakens that they have met on the ocean currents of Norn. Of lost isles filled with forgotten treasure or strange people, or stories even, about the ancient elves who once aided their ancestors and are said to be around still.`

Reviewer 2: ```suggestion
			`	In turn, they tell you about the sea beasts, leviathans, and krakens that they have met on the ocean currents of Norn. Of lost isles filled with forgotten treasure or strange people, or stories even, about the ancient elves who once aided their ancestors and are said to be around still.`
```

			choice
				`	(Ask about the elves.)`

> 					goto elves

Reviewer 2: ```suggestion
```

				`	(Ask about lost treasure.)`
					goto treasure

> 			label elves

Reviewer 2: ```suggestion
```

> 			`	The idea of Elves, or Alfar as they call them, intrigues you and you ask what they know of them.`

Reviewer 2: ```suggestion
			`	The idea of Elves, or Alfar as they call them, intrigues you, and you ask what they know of them.`
```

> 			`	"The Alfar," Thorleif begins, "are said to have been a great help to us in the early days. Some say they spared us from plagues and protected us from monsters. Many still offer sacrifices and prayers to them, so that they remember us and know that we are friends and allies still."`

Reviewer 2: Are the Alfar supposed to be a pseudo-religious interpretation of the original Deep settlers' experience with the Pug? Seems to come off that way, curious if that's the intent.

Author: Yes.

> 			`	Thorleif shrugs, "This may be true, and good at that, but others think the Alfar, for all their apparent good works, cannot be trusted. Their minds are unknowable and filled with mischief, and their aims are not ours. This is the belief that our family holds to. They should be placated, but also feared. We must look to each other for support and strength, and trust not in the deeds of strangers."`

Reviewer 2: ```suggestion
			`	Thorleif shrugs. "This may be true, and good at that, but others think the Alfar, for all their apparent good works, cannot be trusted. Their minds are unknowable and filled with mischief, and their aims are not ours. This is the belief that our family holds to. They should be respected, but also feared. We must look to each other for support and strength, and trust not in the deeds of strangers."`
```
I'd prefer a different word over `placate`, which usually implies the person/thing being placated was hostile - that doesn't seem to be the case with the Alfar.

					goto family
			label treasure
			`	You could always use a little extra money, and this trip, while engaging, has certainly not made you any richer, so you ask about hidden treasures. Helm and Thorleif both grin at this and glance at each other knowingly. They launch into tales of the riches of lost civilizations, or forgotten pirate hoards, or even early human kingdoms. It's easy to lose things on a fog-covered ocean world with floating islands, all too easy.`

> 			`	When they are done regaling you with tales they mention to you that maybe some other time they'll take you out into the deep ocean and perhaps you'll find what you're looking for. They smile wolfishly at that and you think they might be putting you on. "Have you ever found treasure out here?" you ask.`

Reviewer 2: ```suggestion
			`	When they are done regaling you with tales, they mention to you that maybe some other time they'll take you out into the deep ocean and perhaps you'll find what you're looking for. They smile wolfishly at that, and you think they might be putting you on. "Have you ever found treasure out here?" you ask.`
```

> 			`	Thorleif shrugs, "coming here.. from Alfheim long ago, our descendants felt that the world itself was the real treasure. We have no need to go tracing dreams in the abyss."`

Reviewer 2: ```suggestion
			`	Thorleif shrugs. "Coming here... from Alfheim long ago, our descendants felt that the world itself was the real treasure. We have no need to go chasing dreams in the abyss."`
```
It feels like `chasing` might fit better than `tracing` here. Any particular reason you used `tracing`?

Author: No, I probably meant chasing and wrote tracing.

			label family
			`	As the conversation winds down and the hour grows late Helm excuses himself to bed, for he, like the rest of you, is bone tired after the day's events. You stand too, feeling your own need for sleep.`

> 			`	"<first>," Thorleif says without looking up from the fire. "I've been thinking. I can never forgive him for what he did. Leaving me and mom like that. But I can, I think, make peace with it. I can accept the stone." He looks up and smiles, "You brought some luck with you, and a man should never argue with luck."`

Reviewer 2: ```suggestion
			`	"<first>," Thorleif says without looking up from the fire. "I've been thinking. I can never forgive my father for what he did. Leaving me and mom like that. But I can, I think, make peace with it. I can accept the stone." He looks up and smiles, "You brought some luck with you, and a man should never argue with luck."`
```
Just to ensure an abundance of clarity :)

> 			`	He stands up, "We'll talk in the morning."`

Reviewer 2: ```suggestion
			`	He stands up. "We'll talk in the morning."`
```

			choice
				`	(You head to bed for some well deserved sleep.)`

> 			`	The day is bright and the sky is clear. The storm long forgotten as you prepare your things. Out in the main room Thorleif and Helm are waiting for you, and Thorleif opens up his hand to accept the stone, which you gladly place into it. He smiles and looks it over, tracing his finger along its chiseled runes.`

Reviewer 2: ```suggestion
			`	The day is bright and the sky is clear, the storm long forgotten as you prepare your things. Out in the main room, Thorleif and Helm are waiting for you, and Thorleif opens up his hand to accept the stone, which you gladly place into it. He smiles and looks it over, tracing his finger along its chiseled runes.`
```
The second sentence is incomplete, so we'll just throw it into the first sentence to resolve it. I think the flow of the middle part could be improved, so here's my suggestion:
```suggestion
			`	The day is bright and the sky is clear, the storm long forgotten as you prepare your things. Both men meet you in the main room. Thorleif opens up his hand to accept the stone, which you gladly place into it. He smiles and looks it over, tracing his finger along its chiseled runes.`
```

> 			`	"Thank you <first>". You nod, glad to be done with this mission and free to ply the space lanes once more.`

Reviewer 2: ```suggestion
			`	"Thank you, <first>". You nod, happy to be done with this mission and free to fly the space lanes once more.`
```
`Glad` is used in the previous line.

> 			`	"Just more thing," Thorleif adds with a big stupid grin, "I'll need a captain willing to ferry me to Alfheim. That's where my family's carin is located."`

Reviewer 2: ```suggestion
			`	"Just more thing," Thorleif adds with a big, stupid grin. "I'll need a captain willing to ferry me to Alfheim. That's where my family's carin is located."`
```
Just corrections above, my suggestion below.
```suggestion
			`	"Just more thing," Thorleif adds with a sly grin. "I'll need a captain willing to ferry me to Alfheim. That's where my family's carin is located."`
```
`big, stupid` just doesn't seem to match Thorleif's characterization previously to me.

			choice

> 				`	(After all you've been through you don't see any way to refuse the request.)`

Reviewer 2: ```suggestion
				`	(After all you've been through, you don't see any way to refuse the request.)`
```

			`	"Of course," you answer, and before you know it the three of you are on Helm's boat and heading for the spaceport.`
			`	"Alfheim," Thorleif says idly, "you know of it? The name means the home of the Alfar, or elves as you call them. I've never been there, only heard of it. It'll be strange to visit the world of my ancestors."`
			`	Helm stays behind as you and Thorleif load your gear onto your ship.`
			`	You're not sure exactly how you got mixed up into all this, and yet, this all feels right to you. Like you are where you should be, doing what you should do. It might not be intergalactic wars, or the secrets of the ancients, but sometimes it really is just about people. People like Skaldgar and Thorleif.`
				accept
	on visit

> 		dialog `You have reached <planet>, but your escort with the space reserved for Thorleif and Helm has not arrived! Better depart and wait for your escorts to arrive in this star system.`

Reviewer 2: ```suggestion
		dialog `You have reached <planet>, but your escort carrying Thorleif and Helm has not arrived! Better depart and wait for your escorts to arrive in this star system.`
```

			`	Thorleif shakes his head. "Lady, no. We're not.. anyone's employees, nor do we need work here. I am looking for a village, hamlet really, called Grimssalr. It's where my folk come from."`
			`	The woman frowns for a second then seems to break from a trance as real warmth washes over her face. "I am sorry, it's been so long since I've dealt with people who weren't just looking for work. Alfheim, well, it is what it used to be, but only just more so now." She shrugs, "I know of Grimssalr though. I'm actually from Lindsvoll which is in the same valley."`
			`	"So can you tell us where it is?" Thorleif asks earnestly. The woman smiles shyly at Thorleif. "Tell you? How about I take you there?"`
			choice
				`	(She can come.)`
					goto accompany
				`	(We don't need her.)`
			`	You start to say that it isn't necessary for her to accompany you, when Thorleif cuts you off. "That sounds great!"`
			`	She smiles broadly at Thorleif and nods.`
			choice
				`	(You sigh and shake your head.)`
			label accompany

> 			`	She immediately abandons her kiosk and shrugs, "I didn't care much for that job anyways."`

Reviewer 2: ```suggestion
			`	She immediately abandons her kiosk and shrugs. "I didn't care much for that job anyways."`
```
Oops, missed this too.

			choice
				`	(Ask about the elves.)`
					goto elves
				`	(Ask about lost treasure.)`
					goto treasure
			label elves
			`	The idea of Elves, or Alfar as they call them, intrigues you and you ask what they know of them.`
			`	"The Alfar," Thorleif begins, "are said to have been a great help to us in the early days. Some say they spared us from plagues and protected us from monsters. Many still offer sacrifices and prayers to them, so that they remember us and know that we are friends and allies still."`
			`	Thorleif shrugs, "This may be true, and good at that, but others think the Alfar, for all their apparent good works, cannot be trusted. Their minds are unknowable and filled with mischief, and their aims are not ours. This is the belief that our family holds to. They should be placated, but also feared. We must look to each other for support and strength, and trust not in the deeds of strangers."`

> 					goto family

Reviewer 3: Where does this goto go? I don't see a `label family` anywhere?

Author: Ah, a previous reviewer removed that.
I know hot to fix it. But I don't know how to navigate my files on github very well.

Reviewer 3: Looks like you have it fixed now. 👍

			`	"I figured for you to be returning, it would either be with Skaldgar or with" - he pauses - "news." He looks around. "And I don't see Skaldgar."`
			`	You show him the odd stone you'd been given, and Helm stiffens then nods. "As I figured. Best we meet in the spaceport to discuss what comes next."`

mission "Stone of our Fathers 4"
	name "Stone of our Fathers"
	description "Take Skaldgar's stone back to his Family-Cairn on <destination>."
	minor
	passengers 1
	to offer
		has "Stone of our Fathers 3: done"
	source "Norn"
	destination "Alfheim"
	on offer
		conversation
			`Helm meets you in the nearly deserted spaceport bar. "Ma always had a soft spot for Uncle Skaldgar. Even after he ran off 35 years ago, she still spoke fondly of him. He always sent the family money, he was good like that. Not as good, though, in sending his love."`
			`	Helm sighs, "This is going to be tough, but as bearer of the Stone, you are going to need to present it to his son. His family lives on a drifting algae island to the south of here. Are you willing to accompany me?"`
			choice
				`	"I've come this far. I might as well finish it."`
					goto accept

> 				`	"Absolutely, I want to see this through."

Reviewer 4: ```suggestion
				`	"Absolutely, I want to see this through."`
```

			choice
				`	(Head off to bed as the first rains begin to patter against the roof.)`
			`	You awake to the riotous din of hard rain crashing against the building. As Helm predicted, a powerful storm blew in during the night, the floating island besieged by savage weather. By midmorning, the walls are starting to creak as howling forces of nature bash against it unceasingly.`
			`	You hear a door open and struggle to close, followed by the sound of heavy sloshing footsteps. Thorleif, soaked to the bone, stands before you and Helm in the main room. "It's a Fimbulhrithr," Thorleif exclaims, "One of the worst I've yet seen. If we do not act quickly it could well flatten everything on this isle. Will you help me?"`
			choice
				`	(You offer to help Thorleif.)`
					goto help
				`	"No, I've done enough for this family already. Especially for free!"`
			`	Thorleif growls, "Knew better than to trust a friend of that useless old man. I gave you a place to sleep, and now you won't lift a hand to save it? I should have expected no less!" Thorleif and Helm hurry off to tend to the island during the storm, leaving you idle inside.`

> 			`	Thorleif and Helm heroically use the islands metal rudder and rudimentry engine to steer it away from the storm and to calmer, safer seas. However, with only the two of them many of the out-buildings are destroyed, along with a few of the small watercraft Thorleif used to fish and work on.`

Reviewer 2: ```suggestion
			`	Thorleif and Helm heroically use the island's metal rudder and rudimentary engine to steer it away from the storm and to calmer, safer seas. However, with only the two of them many of the out-buildings are destroyed, along with a few of the small watercraft Thorleif used to fish and work on.`
```

			`	"So can you tell us where it is?" Thorleif asks earnestly. The woman smiles shyly at Thorleif. "Tell you? How about I take you there?"`
			choice
				`	(Invite her with you.)`
					goto accompany
				`	(Reject the offer.)`
			`	You start to say that it isn't necessary for her to accompany you, when Thorleif cuts you off. "That sounds great!"`
			`	She smiles broadly at Thorleif and nods.`
			choice
				`	(Sigh and shake your head.)`
			label accompany
			`	She immediately abandons her kiosk and shrugs. "I didn't care much for that job anyways."`
			`	You and Thorleif follow her to a local transit hovercraft, where she and Thorleif quickly sit together. You find yourself sitting alone.`
			`	For the next three hours, it sounds as if they are in deep conversation, while you drift in and out of sleep watching as oil fields give way to rainforests and then rainforests shrink before great wooded mountains.`
			`	At some point you feel Thorleif shaking your shoulder. "We're here."`
			choice

> 				`	(You rise from your seat.)`

Reviewer 2: ```suggestion
				`	(Rise from your seat.)`
```

			`	The woman, who you've learned is named Ragnhild, introduces Thorleif to some of the folks here, and before long they have found ways to connect their ancient genealogies together and everyone is suddenly treated as family. Before you know it, you are eating their meager yet hearty fare, swapping stories of adventures, and listening to legends about the old times.`
			`	As the sun begins to set, Thorleif approaches you, "It is time," he says. "The journey to the cairn is only a short hike from here. I would like you to accompany me. I think Skaldgar would have wanted that."`	
			choice
				`	(Accompany Thorleif.)`
					goto followcairn
				`	"It's best that you do this alone."`
			`	Thorleif frowns. "I cannot. It does not feel right. It is only because of you, that I am here at all. I need you by my side to finish this. For me, and for him."`
			choice
				`	(Accompany Thorleif.)`
					goto followcairn

> 				`	"I am sorry. But I will not go."

Reviewer 4: ```suggestion
				`	"I am sorry. But I will not go."`
```

	description "Travel to <destination> and drop Skaldgar off at the home he abandoned long ago."
	minor
	passengers 1
	to offer
		random < 50
	source "Helheim"
	destination "Norn"
	on offer
		conversation
			`Helheim's spaceport has all the constant hustle and bustle of a boomtown. It's dirty, lively, and packed with workers. As you walk through the spaceport, an aged man in a worn and weathered roustabout jacket waves his arm toward you and approaches you.`
			`	"You've got the look of one who knows a ship inside and out," he says. "Name's Skaldgar, ain't much for ships; earth, soil, and rock are my domain. But I do got a need to travel." He pauses as a wistful look briefly passes over his face. "I left, you see, decades ago. No goodbye. No note. Just... left. Living with others, taking care of others, hard on a man like me. I like solitude, peace, and work. But now my bones are brittle: the radiation, earth-rot as we call it, has taken hold and my time is running out. And I find, I don't want to be alone anymore."`
			`	He stops and frowns. "Look at me, rambling like an ol' man. Ser, Captain, I'd like to pay you <payment> to take me home, to <destination>."`
			choice

> 				`	"If you have coin, I have room."`

Author: Alright, added another reply.

			choice
				`	(You reach out and accept the stone.)`
					goto accept
				`	"Nope. Not going to do it. I run a ship, not a charity."`
			`	The young roustabout nods knowingly. "Guess Skaldgar was wrong about you, but I understand. We all gotta eat." He turns and slides back into the endless crowd.`
				decline
			label accept
			`	You're surprised at how cool the smooth stone feels in your palm.`
			`	He smiles sadly. "Skaldgar always had a nose for good folks. Anyways, he told me he had a nephew named Helm who worked at the Norn spaceport. Find 'em and he can getcha and this stone where it needs to go." The young man than looks you in the eye and gives a slow respectful nod, "For Ol' Skaldgar, may he be findin' ore in hel."`
			`	And with that, the man disappears into the crowd.`
				accept
	on complete
		dialog `It turns out there is no need to search for Helm, as he is waiting for you when you land.`

> 			`	"I figured for you to be returning, it would either be with Skaldgar or with" —he pauses— " news." He looks around. "And I don't see Skaldgar."`

Reviewer 2: I think the intent is "if you're returning, then it would be either with Skaldgar or news," and there's just been words removed to fit the character's speech patterns, in which case the period would change its meaning. Thoughts on keeping it in?
```suggestion
			`	"I figured for you to be returning, it would either be with Skaldgar or with" - he pauses - "news." He looks around. "And I don't see Skaldgar."`
```

			`	On landing, you are surprised by how much Alfheim reminds you of Helheim. Though the environment and ecology of the worlds are remarkably different, both are primarily used for resource extraction, and thus filled with boomtown employees and roustabouts, looking for work, chasing dreams, and trying to make their way in the world.`
			`	Thorleif points to the spaceport. "Let's see if we can find a guide in there. There's a village we'll need to find, assuming it even exists anymore and hasn't been abandoned or destroyed to make room for a new oil rig."`
	on offer
		conversation
			`Helm meets you in the nearly deserted spaceport bar. "Ma' always had a soft spot for Uncle Skaldgar, even after he ran off 35 years ago, she still spoke fondly of him. He always sent the family money, he was good like that. Not as good, though, in sending his love."`
			`	Helm sighs, "This is going to be tough, but as bearer of the Stone, you are going to need to present it to his son. His family lives on a drifting algae island to the south of here. Are you willing to accompany me?"`
			choice

> 				`	"I've come this far. I might as well finish it."`

Author: I have added another reply.

					goto earth
			label from
			`	Helm shrugs as he maneuvers the ship into dock. "Wherever makes the most sense at the time a stone is given. If I recall correctly Skaldgar's stone was given to him by his mother, my grandmother, from an old mine Skaldgar had been exploring on the mainland. Even at 16 he loved the earth."`
			label earth
			`	Helm ties the ship up and you follow him toward a grey and dour building not far from the makeshift dock. You realize this entire algae island, as small as it is, must belong to a single individual or family.`
			`	When you arrive at the hardy and sea-stained wooden door, Helm knocks thunderously upon it and calls out, "Thorleif, open up!"`
			`	After a brief wait the door opens ponderously, and there stands a large, red-bearded, middle aged man who looks like a younger and much healthier Skaldgar. A wide and open grin forms on his face when he sees Helm, and he gives the man a warm embrace, "Ah cousin! It's been too long, what brings you to my humble home?"`
			`	Thorlief then notices you and frowns quizzically. "And who have you brought with you?"`
			choice
				`	(Answer for yourself.)`

> 					goto yourself

Reviewer 2: ```suggestion
```
This will just lead to what's directly after the `choice` without labels needed.

				`	(Let Helm Answer for you.)`
					goto helm

> 			label yourself

Reviewer 2: ```suggestion
```

			`	You smile somberly at Thorleif. "I was a friend of Skaldgar. I gave him a ride a few times, and, most recently, was entrusted with his stone."`
					goto stone
			label helm
			`	Helm puts a hand on Thorleif's shoulder. "This here is <first>, they've come a long way to see you, and to bring you Skaldgar's stone."`
			label stone
			`	Thorleif tenses up. "Never speak that name!" His eyes narrow. "Helm, I want nothing to do with that stone, this person," Thorleif says, gesturing toward you, "or whatever else that old man wanted or cared about."`
			`	Thorleif glances toward the horizon as he clenches and unclenches his fists, taking a few slow breaths, regaining his composure. "I realize you have traveled far and would not be able to get back to the mainland before night. So you may stay in the guest quarters, and leave first thing in the morning. I will have no one say that Thorleif was inhospitable."`
			choice
				`	(Keep quiet and follow Helm towards the guest rooms.)`
					goto rooms
				`	(Try to say something on Skaldgar's behalf.)`

> 					goto behalf
> 			label behalf

Reviewer 2: ```suggestion
```

> 			`	"Wait," you say before Thorleif can stomp off. "I do not, could not, know all that has happened here, that has gone on between you and your father. But I know that every time I met him he clearly regretted whatever happened, and constantly spoke of his mistakes. He is gone now, mistakes and all, and this stone.. I don't think it represents forgiveness, just, acceptance of who he was and that now he is gone."`

Author: I have changed the text along the lines of your suggestion.

			choice
				`	(You head to bed as the first rains begin to patter against the roof.)`
			`	You awake to riotous rain pelting hard against the walls. As Helm predicted, a powerful storm blew in over the night, and the floating island found itself besieged by savage weather. By midmorning the walls are starting to creak as the howling wind bashes against it unceasingly.`
			`	You hear a door open and struggle to close, followed by the sound of heavy sloshing footsteps. Thorleif, soaked to the bone, stands before you and Helm in the main room. "It's a Fimbulhrithr," Thorleif exclaims, "One of the worst I've yet seen. If we do not act quickly it could well flatten everything on this isle. Will you help me?"`
			choice
				`	(You offer to help Thorleif.)`
					goto help
				`	"No, I am done too much for this family already. Especially for free!"`
			`	Thorleif growls, "Knew better than to trust a friend of that useless old man. I gave you a place to sleep, and now you won't lift a hand to save it? I should have expected no less!" Thorleif and Helm hurry off to tend to the island during the storm, leaving you warm and cozy and idle inside.`

> 			`	Thorleif and Helm heroically manage to steer the island away from the storm and to calmer, safer seas. However, with only the two of them many of the out-buildings are destroyed, along with a few of the small watercraft Thorleif used to fish and work on.`

Author: I figure. It seems unlikely that the human inhabitants would not rig up some way to control where their floating-drifting island-homes are headed. I imagine the characters in this mission as mostly rustic-poor, so whatever they have rigged would likely be low-tech and easy to fix/maintain.

> 			`	Thorleif and Helm heroically manage to steer the island away from the storm and to calmer, safer seas. However, with only the two of them many of the out-buildings are destroyed, along with a few of the small watercraft Thorleif used to fish and work on.`

Reviewer 5: This needs to be explained, then. :)

> 			`	Thorleif and Helm heroically manage to steer the island away from the storm and to calmer, safer seas. However, with only the two of them many of the out-buildings are destroyed, along with a few of the small watercraft Thorleif used to fish and work on.`

Author: I added a bit about a metal rudder and a rudimentary engine.

			`	Thorleif shakes his head. "Lady, no. We're not.. anyone's employees, nor do we need work here. I am looking for a village, hamlet really, called Grimssalr. It's where my folk come from."`
			`	The woman frowns for a second then seems to break from a trance as real warmth washes over her face. "I am sorry, it's been so long since I've dealt with people who weren't just looking for work. Alfheim, well, it is what it used to be, but only just more so now." She shrugs, "I know of Grimssalr though. I'm actually from Lindsvoll which is in the same valley."`
			`	"So can you tell us where it is?" Thorleif asks earnestly. The woman smiles shyly at Thorleif. "Tell you? How about I take you there?"`
			choice
				`	(She can come.)`
					goto accompany
				`	(We don't need her.)`
			`	You start to say that it isn't necessary for her to accompany you, when Thorleif cuts you off. "That sounds great!"`
			`	She smiles broadly at Thorleif and nods.`
			choice

> 				`	(You sigh and shake your head.)`

Author: In my play-tests, I liked that it broke up the text,

			label accompany
			`	She immediately abandons her kiosk and shrugs. "I didn't care much for that job anyways."`
			`	You and Thorleif follow her to a local transit hovercraft, where she and Thorleif quickly sit together. You find yourself sitting alone.`
			`	For the next three hours, it sounds as if they are in deep conversation, while you drift in and out of sleep watching as oil fields give way to rainforests and then rainforests shrink before great and arid mountains.`
			`	At some point you feel Thorleif shaking your shoulder. "We're here."`
			choice

> 				`	(Yawning, you rise from your warm seat.)`

Author: Same as before, it was something that in my play-tests felt good. Breaking up the texts and giving the player something active (if only clicking on a single choice) to do felt better.

> 				`	(Yawning, you rise from your warm seat.)`

Reviewer 5: I can understand that, but then this needs to be rephrased in the imperative. But using the imperative form of the verb "to yawn" is a bit odd lol

> 				`	(Yawning, you rise from your warm seat.)`

Author: I suppose you wouldn't like, "You yawn and rise.."?

> 				`	(Yawning, you rise from your warm seat.)`

Reviewer 2: You can't really use `you` like this in the [imperative](https://grammar.collinsdictionary.com/us/easy-learning/the-imperative). One example would be `Rise from your seat.` Or maybe `Rise from your seat, yawning.`?

> 				`	(Yawning, you rise from your warm seat.)`

Author: Guess I'll go with "Rise from your seat"

			`	The town of Grimssalr is nestled along the foothills of the towering Ettin mountains in western Alfheim. Town is perhaps too generous a word; it is instead more like a couple dozen houses mostly filled with old people long abandoned by their children seeking wealth and adventure elsewhere.`
			`	The woman, who you've learnt is named Ragnhild, introduces Thorleif to some of the folks here, and before long they have found ways to connect their ancient genealogies together and everyone is suddenly treated as family. You eat their meager fare, swap stories, and listen to legends about the old times.`
			`	As the sun begins to set, Thorleif approaches you, "It is time," he says. "The journey to the cairn is only a short hike from here. I would that you accompany me. I think Skaldgar would have wanted that."`	
			choice
				`	(Leave Grimssalr with Thorleif.)`
					goto followcairn
				`	"It is best you do this alone."`

> 					goto stayingrim
> 			label stayingrim

Reviewer 2: ```suggestion
```

> 			`	Thorleif frowns. "I cannot. It does not feel right. It is only because of you, that I am here at all. I need you by my side to finish this. For me, and for him.`

Reviewer 2: ```suggestion
			`	Thorleif frowns. "I cannot. It does not feel right. It is only because of you, that I am here at all. I need you by my side to finish this. For me, and for him."`
```

			choice
				`	(Leave with Grimssalr with Thorleif.)`
					goto followcairn
				`	"I am sorry. But I will not go."

> 					goto beingstupid
> 			label beingstupid

Reviewer 2: ```suggestion
```

> 			`	Thorleif's face reddens and abruptly he turns from you. He has no words left to argue with and leaves you to wait in the village.`

Reviewer 2: ```suggestion
			`	Thorleif's face reddens and he abruptly turns from you. He has no words left to argue with and leaves you to wait in the village.`
```

			`	A short time later he returns, his eyes wet and red.`
			`	"I have made my peace, I have read the runes of his life and returned his stone to the Family-Cairn of Hlewagast. He had few happy moments, and the only recent ones involved us both. He is gone now, and we remain." Thorleif wipes tears from his eyes. "Let us rest, and eat, and drink for a time. Skaldgar would have liked that, too."`
				goto drinkandbemerry
			label followcairn
			`	You follow Thorleif up a mostly overgrown trail for about 30 minutes before finally stepping out into a wide open plain. The wind is cool and refreshing, and the grass blows with a steadiness that is somehow reassuring. There is something special, timeless, about this sudden meadow.`

> 			`	You notice over two dozen columns of lain stone arranged in a circle around a much more massive and impressive structure. The lain stone columns must be cairns of various nearby families, but the massive structure in the center is something completely different. The structure is a wall, cut from some sort of metal you don't recognize and filled with letters and images that are unfamiliar to you.`

Author: Added descriptor to the metal.

			`	"The day my mother gave me this stone and said she was proud of me."`
			`	"The day my son Thorleif was born."`
			`	"The day I struck the mother lode on Helheim."`
			`	"The day I met <first> and he helped me find my family again."`
			`	"The day I saw my son again, even though he would not see me."`
			`	Thorleif sniffs, takes a moment to write something down he saw on the stone, and then walks forward and places it upon the cairn, at home now with hundreds of other stones each with their own handful of memories.`
			`	"He had not so many good memories, I think," Thorleif said quietly. "I realize now, that mine too are quite few. Perhaps that, if nothing else, is a lesson he can teach me. For that, I thank you, dad."`
			`	With that, Thorleif turns from the cairn with blurry eyes. "It is done. Let us head back."`
			choice
				`	(Follow Thorleif back to the village.)`	
			`	The two of you trudge back to the small hamlet of Grimssalr, with its people happy at having guests and long lost relatives, and to Ragnhild, happy to spend more time with Thorlief, where you feast and talk and laugh and remember.`

> 					goto drinkandbemerry

Reviewer 2: ```suggestion
```

			label drinkandbemerry
			`	You enjoy yourself well into the evening before Thorleif approaches you. It's time to head back.`
			`	"This place," Thorleif begins, "this world, it feels like it has forgotten its soul. Industry and greed have taken hold, and the old happy places like Grimssalr are dying. It is good my family left here all those years ago. Norn is, and has been, a good place to be."`
			`	Thorleif then nods to Ragnhild. "Which is also why my very new and very dear friend Ragnhild has decided to join us on the return trip to Norn." You look to Ragnhild but somehow you can't feel that you're surprised at the news.`
			`	She smiles brightly. "Best offer I've had in a long time." The three of you board your ship and prepare for the journey home.`
				accept
	on visit
		dialog `You have reached <planet>, but your escort with the space reserved for Thorleif and Ragnhild has not arrived! Better depart and wait for your escorts to arrive in this star system.`
	on complete
		payment 500000
		conversation 
			`The trip back to Norn was restful.`

> 			`	You spent most of it listening to Ragnhild tell the old stories and legends of Alfheim that she had learnt in her girlhood.`

Reviewer 5: ```suggestion
			`	You spent most of it listening to Ragnhild tell the old stories and legends of Alfheim that she had learned as a child.`
```

> 			`	Stories about alien trees whispering dark secrets, of Oil-Barons destroying sacred sites and being cursed by alien artifacts they were too blinded by greed to see, and even tales of the elves and miracles of the Elf-Stones.`

Reviewer 5: ```suggestion
			`	Stories about alien trees whispering dark secrets, of oil barons destroying sacred sites and being cursed by alien artifacts they were too blinded by greed to see, and even tales of the elves and miracles of the Elf-Stones.`
```

		add fleet "Small Northern Pirates" 400
		add fleet "Large Northern Pirates" 500
		add fleet "Large Republic" 15000

mission "Stone of our Fathers 1"
	name "Stone of our Fathers"

> 	description "Travel to <destination> and drop Skaldgar off at the home he abandoned long ago."

Reviewer 2: Any reason why this was left uncommitted?

			choice
				`	(You reach out and accept the stone)`
					goto accept
				`	"Nope. Not going to do it. I run a ship not a charity."`
			`	The young roustabout nods knowingly. "Guess Skaldgar was wrong about you, but I understand. We all gotta eat." He turns and slides back into the endless crowd.`
				decline
			label accept
			`	You're surprised at how cool the smooth stone feels in your palm.`
			`	He smiles sadly, "Skaldgar always had a nose for good folks. Anyways, he told me he had a nephew named Helm who worked at the Norn spaceport. Find 'em and he can getcha and this stone where it needs to go." The young man than looks you in the eye and gives a slow respectful nod, "For Ol' Skaldgar, may he be findin' ore in hel."`
			`	And with that the man disappears into the crowd.`
				accept
	on visit
		dialog `You have reached <planet>, but your escort with the space reserved for the stone has not arrived! Better depart and wait for your escorts to arrive in this star system.`
	on complete
		payment 15

> 		dialog `It turns out there is no need to search for Helm, as he is waiting for you when you land.`

Author: Yeah, that has worked in my play-tests.

			`	"I figured for you to be returning, it would either be with Skaldgar or with," he pauses," news." He looks around, "and I don't see Skaldgar."`
			`	You show him the odd stone you'd been given and Helm stiffens then nods. "As I figured, best we meet in the spaceport to discuss what comes next."`

mission "Stone of our Fathers 4"
	name "Stone of our Fathers"
	description "Take Skaldgar's stone back to his family carin <destination>."
	minor
	cargo "Family stone" 1
	to offer
		has "Stone of our Fathers 3: done"
	source "Norn"
	destination "Alfheim"

> 	on complete

Author: I have included the option at 2708 to not go to the Family-Cairn. I will wait for you to look at that new section and make edits. After those edits are made that should resolve all lingering edits, save this one, and so I will make this and hope that the devs find the mission pleasing.

> 	on complete

Author: Alright, the on complete now happens after the on offer.

			`	Through all of this you got a better feel for Thorleif, Skaldgard and his people, and were almost sorry when your ship approached Alfheim.`
			`	On landing you are surprised by how much Alfheim reminds you of Helheim. Though the environment and ecology of the worlds are remarkably different, both are primarily used for resource extraction. Filled with boomtown employees and roustabouts, looking for work, chasing dreams, and trying to make their way in the world.`
			`	Thorleif points to the Spaceport, "Lets see if we can find a guide in there. There's a village we'll need to find, assuming it even exists anymore and hasn't been abandoned or destroyed to make room for a new oil rig."`
	on offer
		conversation
			`Helm meets you in the nearly deserted Spaceport bar. "Ma' always had a soft spot for Uncle Skaldgar, even after he ran off 35 years ago, she still spoke fondly of him. He always sent the family money, he was good like that. Not as good, though, in sending his love."`

> 			`	Helm sighs, "This is going to be tough, but as bearer of the Stone, you are going to need to present it to his son. His family lives on a floating island to the south of here. Are you willing to accompany me?"`

Author: On the planet Norn, there apparently are regular rock (non-floating) and algae (floating) islands.

			choice
				`	(Keep quiet and follow Helm towards the guest rooms.)`
					goto rooms
				`	(Try to say something on Skaldgar's behalf.)`
					goto behalf
			label behalf
			`	"Wait," you say before Thorleif can stomp off, "I do not, could not, know all that has happened here, that has gone on between you and your father. But I know that every time I met him he clearly regretted whatever happened, and constantly spoke of his mistakes. He is gone now, mistakes and all, and this stone.. I don't think it represents forgiveness, just, acceptance of who he was and that now he is gone."`
			`	Thorleif turns to you, his face flashing red, and speaks through clenched teeth, "I have no father. I have no memories of such a man, and I will make no effort to remember someone who didn't stick around long enough to know me. Now, go to your rooms and leave me be!"`
			label rooms
			`	You quietly follow Helm to two small, sparsely decorated rooms, clearly intended for guest use. Helm clears his throat, "This may surprise you, but that actually went better than I thought it would."`

> 			`	Helm then turns his attention to a small window and the darkening horizon beyond it. "Dark clouds are gathering, I fear we may be here longer than a single night."`

Author: I have spoken with Zitchas and he said it was okay for time to pass this way. Based off the belief that he is right, I have decided to leave it in.

					goto valley
			label valley
			`	"So can you tell us where it is?" Thorleif asks earnestly. The woman smiles shyly at Thorleif, "Tell you? How about I take you there."`
			choice
				`	(She can come.)`
					goto accompany
				`	(We don't need her.)`
					goto noneed
			label noneed
			`	You start to say that it isn't necessary for her to accompany you, when Thorleif cuts you off. "That sounds great!"`
			`	She smiles broadly at Thorleif and nods.`
			choice
				`	(You sigh and shake your head.)`
			label accompany
			`	She immediately abandons her kiosk and shrugs, "I didn't care much for that job anyways."`
			`	You and Thorleif follow her to a local transit hovercraft, where she and Thorleif quickly sit together and you find yourself sitting alone.`
			`	For the next three hours it sounds as if they are in deep conversation, while you drift in and out of sleep watching as oil fields give way to rainforests and then rainforests shrink before great and arid mountains.`
			`	At some point you feel Thorleif shaking your shoulder. "We're here."`
			choice

> 				`	(Yawning you rise from your warm seat)`

Author: I am using it here to break up the text. It also allows for a thematic break from the train scene to the village scene.

> 			choice
> 				`	(Yawning you rise from your warm seat)`

Reviewer 2: ```suggestion
			choice
				`	(Yawning, you rise from your warm seat.)`
```
Sounds good. Can I add these couple edits though?

			label here
			`	The town of Grimssalr is nestled along the foothills of the towering Ettin mountains in western Alfheim. Town, is perhaps too generous a word, it is instead more like a couple dozen houses mostly filled with old people long abandoned by their children seeking wealth and adventure elsewhere.`
			`	The woman, who you've learnt is named Ragnhild, introduces Thorleif to some of the folks here and before long they have found ways to connect their ancient genealogies together and everyone is suddenly treated as family. You eat their meager fare, swap stories, and listen to legends about the old times.`
			`	As the sun begins to set Thorleif approaches you, "It is time," he says. "The journey to the cairn is only a short hike from here. I would that you accompany me. I think Skaldgar would have wanted that."`	
			choice

> 				`	(Leave Grimssalr with Thorleif.)`

Author: I have added the option starting at 2708.

					goto grimfollow
			label grimfollow

> 			`	You follow Thorleif up a mostly overgrown trail for about 30 minutes before finally stepping out into a wide open plain. The wind is cool and refreshing and the grass blows with a steadiness that is somehow reassuring. There is something special, timeless, about this sudden meadow.`

Reviewer 2: Just a quick note: this whole section (Going to the Elf-Stone and adding the stone) is really, really well written. Good stuff, dude!

			`	You notice over two dozen columns of lain stone arranged in a circle around a much more massive and impressive structure. The lain stone columns must be cairns of various nearby families, but the massive structure in the center is something completely different. The structure is a wall, cut from some sort of metal you don't recognize and filled with letters and images that are unfamiliar to you.`
			`	"That is an Elf-Stone. It is why we built our Family-Cairns here. Such stones can be found on most worlds in The Deep. Indestructible, and left by the Alfar for some unknowable reason. Some, like this one, have been mostly forgotten. Others though, are important places of celebration and even worship.`
			`	He points at a smallish cairn next to the Elf-Stone. "This, though, is what we're here for."`
			choice

> 				`	(You eye the worn and weather beaten cairn with a faded name plate reading, 'Hlewagast' carved at its base.)`

Author: As before, my goal here is to keep the text on the screen from becoming too overwhelming. I fear blocks of inscrutable texts. Options like this allow me to keep the text block slightly smaller, and break it up with actions. That, at least, is my reasoning for it.

> 				`	(You eye the worn and weather beaten cairn with a faded name plate reading, 'Hlewagast' carved at its base.)`

Reviewer 2: This would be a spot where I don't think a thematic break is really needed. There's no scene transition, right? And this is essentially the climax of the mission chain, so if there's a spot to have a hefty chunk of text this would be it. That's my take on it, but you do you.

> 				`	(You eye the worn and weather beaten cairn with a faded name plate reading, 'Hlewagast' carved at its base.)`

Author: Alright, sounds good. I will take your suggestion.

					goto thorcarn
			label thorcarn
			`	Thorleif approaches the cairn.`
			`	Thorleif holds up the stone of his father in his palm and begins to read the runes carved upon it.`
			`	"The day my mother gave me this stone and said she was proud of me."`
			`	"The day my son Thorleif was born."`
			`	"The day I struck the mother lode on Helheim."`
			`	"The day I met <first> and he helped me find my family again."`
			`	"The day I saw my son again, even though he would not see me."`
			`	Thorleif sniffs, and takes a moment to write something down he saw on the stone, and then walks forward and places it upon the cairn, at home now with hundreds of other stones each with their own handful of memories.`
			`	"He had not so many good memories, I think," Thorleif said quietly. "I realize now, that mine too are quite few. Perhaps that, if nothing else, is a lesson he can teach me. For that, I thank you, dad."`

> 			`	With that Thorleif turns from the cairn with blurry eyes. "It is done. Let us head back."`

Reviewer 2: ```suggestion
			`	With that, Thorleif turns from the cairn with blurry eyes. "It is done. Let us head back."`
```

			choice

> 				`	(Follow Thorleif back to the village.)`	

Reviewer 2: This would be a spot where I'd say keep it in. This is a good pause since we're going back to the village.

> 			`	The two of you trudge back to the small hamlet of Grimssalr, with its people happy at having guests and long lost relatives, and to Ragnhild, happy to spend more time with Thorlief, where you feast and talk and laugh and remember.`

Reviewer 2: Polysyndeton! Love it!

> 			choice
> 				`	(You enjoy yourself well into the night before finally succumbing to sleep.)`
> 			`	In the morning the three of you catch the hovercraft back to the spaceport.`

Reviewer 2: Yup, this is another spot where we'd need to get rid of the day passing before we return to the ship. This one does have a pretty easy solution though, so much so that I'll suggest it myself.
```suggestion
			`	You enjoy yourself well into the evening before Thorleif approaches you. It's time to head back.`
```
Thoughts? We can play around with this if you want.

> 			`	"This place," Thorleif begins, "this world, it feels like it has forgotten its soul. Industry, and greed have taken hold, and the old happy places like Grimssalr are dying. It is good my family left here all those years ago. Norn is, and has been, a good place to be."`

Reviewer 2: ```suggestion
			`	"This place," Thorleif begins, "this world, it feels like it has forgotten its soul. Industry and greed have taken hold, and the old happy places like Grimssalr are dying. It is good my family left here all those years ago. Norn is, and has been, a good place to be."`
```

> 			`	Thorleif than nods to Ragnhild, "Which is also why my very new and very dear friend Ragnhild has decided to join us on the return trip to Norn." You look to Ragnhild but somehow you can't feel that you're surprised at the news.`

Reviewer 2: ```suggestion
			`	Thorleif then nods to Ragnhild. "Which is also why my very new and very dear friend Ragnhild has decided to join us on the return trip to Norn." You look to Ragnhild but somehow you can't feel that you're surprised at the news.`
```

> 			`	She smiles brightly and shrugs, "Best offer I've had in a long time." The three of you board your ship and prepare for the journey home.`

Reviewer 2: ```suggestion
			`	She smiles brightly. "Best offer I've had in a long time." The three of you board your ship and prepare for the journey home.`
```
You tend to use shrugging a lot, and usually it's fine, but this is probably the one spot where I think maybe leaving it out would be a good idea. I sort of get what you're going for, but by this time it could be repetitive anyways.

				accept
	on visit
		dialog `You have reached <planet>, but your escort with the space reserved for Thorleif and Ragnhild has not arrived! Better depart and wait for your escorts to arrive in this star system.`
	on complete
		payment 500000

> 		conversation 

Reviewer 2: Love this ending as well.

			`The trip back to Norn was restful.`
			`	You spent most of it listening to Ragnhild tell the old stories and legends of Alfheim that she had learnt in her girlhood.`

> 			`	Stories about alien trees whispering dark secrets, of Oil-Barons destroying sacred sites and being cursed by alien artefacts they were too blinded by greed to see, and even tales of the elves and miracles of the Elf-Stones.`

Reviewer 2: ```suggestion
			`	Stories about alien trees whispering dark secrets, of Oil-Barons destroying sacred sites and being cursed by alien artifacts they were too blinded by greed to see, and even tales of the elves and miracles of the Elf-Stones.`
```
British vs. American spelling thing.

			`	Again, as before, you're almost sad when you make it back to Norn and the time for telling stories has ended, and the time for saying goodbye has begun.`

> 			`	Thorleif wraps you in a bear hug. "I thank you for all you have done <first>. I was so wound up in my own anger and bitterness I couldn't see anything beyond my own pain. But now, I feel like I can live again. Trust, that our meeting has been etched into my own stone. I hope someday you come back and visit us. To remember us small folk on a small world so far from places of power and grand events. Remember you'll always have a home here."`

Reviewer 2: ```suggestion
			`	Thorleif wraps you in a bear hug. "I thank you for all you have done <first>. I was so wound up in my own anger and bitterness I couldn't see anything beyond my own pain. But now, I feel like I can live again. Trust that our meeting has been etched into my own stone. I hope someday you come back and visit us. To remember us small folk on a small world so far from places of power and grand events. Remember you'll always have a home here."`
```

> 			`	With that Thorleif and Ragnhild make their way off your ship. Thorleif only takes a few steps out into the quiet Norn Starport, before stopping suddenly and turning back towards you.`

Reviewer 2: ```suggestion
			`	With that Thorleif and Ragnhild make their way off your ship. Thorleif only takes a few steps out into the quiet Norn spaceport before stopping suddenly and turning back towards you.`
```

			`	"I almost forgot. The old man, Skaldgar, well he had quite a bit of money saved up. I don't think he hardly spent anything on himself. Anyways, I've decided to split the inheritance with you. After taxes your half comes out to around 500,000 even."`

> 			`	Thorleif, shrugs, laughs and walks with Ragnhild hand in hand out into the foggy mists of Norn.`

Reviewer 2: ```suggestion
			`	Thorleif, shrugs, laughs, and walks with Ragnhild hand in hand out into the foggy mists of Norn.`
```
